build.go

```diff
@@ -43,7 +44,9 @@ func Build(
 		logger.Title("%s %s", context.BuildpackInfo.Name, context.BuildpackInfo.Version)
 		logger.Process("Resolving Bundler version")
 
-		entry := entries.Resolve(context.Plan.Entries)
+		entry, allEntries := entries.Resolve("bundler", context.Plan.Entries, []interface{}{"BP_BUNDLER_VERSION", "buildpack.yml", "Gemfile.lock", ""})
```

Comment:
You don't need the empty string in the priorities list. Anything that isn't in the list will receive the lowest priority automatically.

Reply:
oops, you're right!